docs/data-cookbooks/cnm-workflow.md

```diff
@@ -476,3 +476,13 @@ Note that the data encoding is not human readable in the stream and would need t
       }
     }
 ```
+
+## Kinesis Stream Error handling
+
```

Comment:
Capitalize Cumulus - Cumulus system. Mention that the kinesisFallback topic, kinesisConsumer lambda, and queue come with the API package and do not need to be configured outside of that. 

Also you can link to dead letter queue: https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-dead-letter-queues.html

And SQS on the AWS console: https://console.aws.amazon.com/sqs/home